src/Avalonia.FuncUI.UnitTests/Program.fs

```diff
@@ -1,6 +0,0 @@
-namespace Avalonia.FuncUI.UnitTests
-
-module Program =
-
-    let [<EntryPoint>] main _ =
```

Comment:
XUnit 3 can run tests directly just by doing 'dotnet run' on the project, which generates the entry points for you

![image](https://github.com/user-attachments/assets/1e07204c-4795-467f-a2d7-9a4d91ccb9c9)